ethereum/test/enlistment_to_contract.js

```diff
@@ -43,18 +43,23 @@ contract('EnlistmentToContract', async ([owner]) => {
     });
 
     it('should instantiate the landlord property', async () => {
-      let landlord = await contract.landlord.call();
+      let landlord = await contract.getLandlord();
```

Comment:
lets agree on a calls and transactions. as truffle suggests, readonly stuff will go via `someMethod.call()`, writeable via `someMethod()`

Reply:
So what you suggest is using `contract.getLandlord.call()` to comply with the Solidity guidlines. Correct? In this case, it's just a syntactic vomit which we should use. PS, could you link to this guideline? I remember discussing it in the early stages but could not find any references online.

Reply:
The initial reasoning behind moving to getter access functions is to restrict access of contract data. E.g. this:
```solidity
-    string public landlord;
-    Enlistment public enlistment;
+    string landlord;
+    Enlistment enlistment;
```
Guideline here: [link](http://solidity.readthedocs.io/en/develop/contracts.html#visibility-and-getters)

Reply:
yes, http://truffleframework.com/docs/getting_started/contracts#executing-contract-functions

Reply:
I am okay with getter access, but truffle suggests to use `call()` when we just read the data, so in this case it will be `contract.getLandlord.call()`

Reply:
Agreed. But let's merge the PRs first and then change the function call style all at once in master.